vars/rpmDashboardsDistValidation.groovy

```diff
@@ -0,0 +1,127 @@
+def call(Map args = [:]) {
+
+    def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    def bundleManifestObj = args.bundleManifestObj
+    def distFile = args.rpmDistribution         //Distribution file location
+    def name = bundleManifestObj.build.getFilename()   //opensearch-dashboards
+    def version = bundleManifestObj.build.version        //1.3.0
+    def architecture = bundleManifestObj.build.architecture
+    def plugin_names = bundleManifestObj.getNames();
+
+    def latestOpenSearchURL = "https://ci.opensearch.org/ci/dbc/Playground/tianleh-test/tianle-opensearch-build-3-22/$version/latest/linux/$architecture/rpm/dist/opensearch/opensearch-$version-linux-${architecture}.rpm"
```

Comment:
Will need changing, externalize the root part of this? (https://ci.opensearch.org/ci/dbc/Playground/tianleh-test/tianle-opensearch-build-3-22)

Reply:
Yes. This is a temporary distribution URL and need to be updated once `distribution-build` running for rpm.